terraform/src/metric.tf

```diff
@@ -0,0 +1,107 @@
+resource "aws_lambda_function" "metricLambda" {
+  function_name = "metric"
+
+  filename         = "artifacts/metric.zip"
+  source_code_hash = filebase64sha256("artifacts/metric.zip")
+
+  handler = "lambda/metric/index.handler"
+  runtime = "nodejs14.x"
+  timeout = 30
+
+  environment {
+    variables = {
+      TABLE_NAME                   = data.aws_ssm_parameter.db_name.value,
+      JWT_SECRET                   = local.jwtSecret.jwtSecret,
```

Comment:
Should be able to remove most of these env vars for this lambda.

Reply:
d3fda1e